arkouda-docker/README.md

```diff
@@ -239,6 +239,26 @@ after the Docker container is stopped. In the above example, the developer write
 container. Once development is complete, the Docker container can be stopped and the updated Chapel files are ready for further development, 
 commit and check-in to github, etc...
 
+## Known Issue and Resolution
+
+Occasionally there are 1..n breaking changes introduced into Arkouda post-release tag that require the arkouda-smp-developer image to be built off the Arkouda master branch. For example, if new Chapel [compat](https://github.com/Bears-R-Us/arkouda/tree/master/src/compat) module(s) are added, the arkouda-smp-developer must be built off the Arkouda master branch.
+
+An example arkouda-smp-developer master branch build is as follows:
+
+```
+export CHAPEL_SMP_IMAGE=bearsrus/chapel-gasnet-smp:1.30.0
```

Comment:
I just wanted to note that `1.31` is out, but I know that we tend to wait a bit to convert the docker images due to issues sometimes. Also I know this is just an example, so it should be expected that the version might need to change